dev/poetry2recipe.py

```diff
@@ -0,0 +1,135 @@
+import io
+import re
+import sys
+from pathlib import Path
+from typing import Optional, Sequence
+
+import tomli
+from poetry.core.factory import Factory
+from poetry.core.masonry.builders.sdist import SdistBuilder
+from ruamel.yaml import YAML
+
+
+def main(
```

Comment:
@cpcloud could you consolidate the two poetry scripts into a single one? 

Eventually I think we should have a python library containing all the required functionality for development. Including the poetry conversion scripts, datamgr, impalamgr and possibly other utilities in the future.

Reply:
Maybe but these are pretty different tools, so I'm not sure putting them into one place makes sense. I think making poetry2recipe a separate package outside of ibis would be useful.

Reply:
After talking with @mariusvniekerk I think we can remove `poetry2recipe` altogether after the next release of ibis, and depend on conda-forge machinery to discover dependencies.